ps_qualityassurance.php

```diff
@@ -81,26 +102,113 @@ public function __call($methodName, array $arguments)
     {
         $hookName = preg_replace('~^hook~', '', $methodName);
 
+        $arguments[4] = ['4' => ['5' => []]];
```

Comment:
I don't understand this line :thinking:

Reply:
Whoops!

Debug code written at 10p 😅  indeed should not be there

Reply:
Fixed in https://github.com/PrestaShop/ps_qualityassurance/pull/8/commits/da03ba8a722b9e6ec2da896b72f65667ddc7c0a4